src/js/components/organisms-page/resellers/resellers.html

```diff
@@ -61,15 +62,23 @@ <h3 class="reseller-section__title">{{ labels.authorizedResellers }} <span
                             </button>
                         </div>
                     </template>
+                </div>
+                <transition-group @before-enter="beforeEnter"
+                                  @enter="enter"
+                                  tag="div"
+                                  :css="false"
+                                  class="flexgrid flexgrid--gutter flexgrid--mobile-xl-row" id="newSection">
                     <div class="flexgrid__cell--md-4 flexgrid__cell--xs-6"
                          v-for="(reseller, index) in resellersAuthorized"
                          v-if="index >= showMax"
-                         v-show="visibleAllAuthorizedResellers">
+                         v-show="visibleAllAuthorizedResellers"
+                         :key="`reseller${index}`"
```

Comment:
Do you really need reseller${index} as key? Wouldn't using the index be quicker?